apps/verify-conformance-release/deployment.yaml

```diff
@@ -0,0 +1,68 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: verify-conformance-release
+  namespace: prow
```

Comment:
Add a new dedicated namespace: https://github.com/kubernetes/k8s.io/blob/fa4da59ac6a42788c2d4bee59916de80f7fc37ca/infra/gcp/bash/namespaces/ensure-namespaces.sh#L145-L161

Reply:
@ameukam, thank you for taking a look!
As the bot interacts with PRs over at cncf/k8s-conformance, will the secrets be available to be authenticated as prow.k8s.io in a new namespace?

https://github.com/kubernetes/k8s.io/pull/6357/files/0b38794494a108fac67e0a8cf78cb969217eddd2#diff-03d8e004cec916a342fd5c6664a9ed4483a62b3ba1804a77854de00d3e94f9ebR56-R65

Reply:
We don't currently have a way to authenticate to `prow.k8s.io` in the `aaa` and we don't have access to the GKE cluster running `prow.k8s.io`.

Reply:
@ameukam, OK this is good to know.

Would think that coordinating with community github-management to create an OAuth app to then provide the credentials to the bot with is a suitable alternative solution?

Reply:
The historical approach of this is to request a token for one of the bots managed by github management team. Not sure how they feel about an OAuth app. Feel free to ask them.

Reply:
I think the historical approach would work just fine, I'll reach out. Thank you!